public/keymaps/coseyfannitutti_discipline_default.json

```
@@ -0,0 +1,22 @@
+{
+  "keyboard": "coseyfannitutti/discipline",
+  "keymap": "default",
```

@coseyfannitutti, you can't actually use `default` as the value for `keymap`, because the API that supports Configurator can see the `keymap.c` files that are in qmk_firmware, even though Configurator itself can't do anything with them. In fact, the value for `keymap` cannot match *any* `keymap.c` that exists in qmk_firmware, for the same reason. The API tries to make a `keyboards/{keyboard}/keymaps/{keymap}/keymap.c` file when the compile job completes, but if one already exists the operation fails.

I append a partial commit hash to the `keymap` value to prevent this. (What you append actually doesn't matter so long as the corresponding file doesn't already exist in qmk_firmware.)

```suggestion
  "keymap": "default_430c370",
```